sci-physics/fastjet/fastjet-3.4.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FORTRAN_NEEDED=plugins
+PYTHON_COMPAT=( python3_{8..10} )
+DOCS_BUILDER="doxygen"
+# we depend on these similar to putting doxygen[dot] in BDEPEND
+DOCS_DEPEND="media-gfx/graphviz media-libs/freetype"
+
+inherit autotools flag-o-matic fortran-2 python-r1 docs
```

Comment:
When we inherit `python-r1` we imply that we want to build the python bindings for multiple enabled python versions (PYTHON_TARGETS). However, we do not do this in the ebuild below (we do not use the `python_foreach_impl` function), this would result in only building the python bindings for the highest enabled PYTHON_TARGET which is not what we want to do when we use `python-r1`. The easy solution (rather then using `python_foreach_impl` in prepare/configure/compile/install) is to just use `python-single-r1` instead, which is what I changed before merging. If we want we can always enable support for multiple simultaneous python versions at some later time, but I don't think it is necessary for this package.